.krew.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: kubectl-flyte
+spec:
+  version: "{{ .TagName }}"
+  homepage: https://github.com/lyft/flytepropeller
+  shortDescription: >-
```

Comment:
`kubectl kubectl-flyte` is not a right way to communicate can we use some other name. Because when user is using it with kubectl then intend is clear, we don't need kubectl to specify the intend

Reply:
my understanding is kubectl uses this convention to "discover" plugins, the binary should be called `kubectl-SOMETHING` and it then allows you to call `kubectl SOMETHING get... ` so the usage description here should say `kubectl flyte get -n my-namespace` to retrieve workflows.... etc.